sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultPodSpec.java

```diff
@@ -25,7 +25,7 @@
     @NotNull
     @Size(min = 1)
     private final String type;
-    private final String user;
+    private String user;
```

Comment:
actually, is there an annotation where we can have a default value of `root` when `user` is unset?

https://stackoverflow.com/questions/29176325/give-a-default-value-for-an-attribute-if-the-value-is-null-in-json-by-jackson